resources/com.sap.piper/pipeline/cloudSdkStageDefaults.yml

```diff
@@ -36,6 +40,10 @@ stages:
       healthExecuteCheck:
         configKeys:
           - 'testServerUrl'
+      npmExecuteEndToEndTests:
+        filePattern: 'package.json'
+        configKeys:
+          - 'appUrls'
```

Comment:
Could you please check whether the conjunction between these conditions is "and" or "or". Because if it is or the end to end tests would run everytime there is a package.json. I think appUrls is anyway sufficient.

Reply:
I will check. Ok I wasn't sure since we were checking for both conditions before.